### PR TITLE
Update yotpo-reviews-intro.md

### PR DESCRIPTION
**Pull request by abigailyotpo**
## Purpose of this pull request

Update for Magento 2.4.1 - we have a new Yotpo enhancement

This pull request (PR) ...

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/yotpo-reviews-intro.html

whatsnew
[Yotpo Reviews](https://docs.magento.com/user-guide/marketing/yotpo-reviews-intro.html#changes-in-the-latest-release) module enhancement to programmatically  adjust when Magento orders are synced to Yotpo.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/marketing/yotpo-reviews-intro.md [hunk 1]
```diff
@@ -19,16 +19,6 @@ With Magento v2.3.3+, integrating your Magento store is a quick, simple process
...
-- <!---Bundle-2422--->Yotpo star ratings can now be displayed on pages created by Magento [Page Builder](https://devdocs.magento.com/page-builder/docs/).
-
-- <!---Bundle-2551--->To enable customers to view and resolve Yotpo-related issues, a new dedicated Yotpo log file (yotpo_yotpo.log) has been added.
-
-### Resolved issues
-
-- <!---Bundle-2549--->To correct errors syncing product brand data, added support for a brand attribute with the Catalog Input Type defined as drop-down menu.
-
-- <!---Bundle-2548--->Order data sync has been changed to no longer send data for canceled items to Yotpo.
+- To enable customers to adjust when orders are synced to Yotpo, the following Yotpo extension functions have now been made programmatically accessible: getOrderCollection, setOrderCollectionLimit, and addOrderCollection.
```
Comment (hickskenh) [inline src/marketing/yotpo-reviews-intro.md:24]:
```suggestion
- The following Yotpo extension functions are now programmatically accessible: `getOrderCollection`, `setOrderCollectionLimit`, and `addOrderCollection`. This enhancement enables customers to adjust when Magento orders are synced to Yotpo.
```

**Review by hickskenh — CHANGES_REQUESTED** [review]

**Review by hickskenh — DISMISSED** [review]

**Comment by hickskenh** [comment]
@abigailyotpo review my suggested edit and commit.  Also, don't forget to sign the CLA.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**